configure.ac

```diff
@@ -332,6 +344,14 @@ PKG_CHECK_MODULES(OSTREE, [ostree-1 >= $OSTREE_REQS])
 PKG_CHECK_MODULES(JSON, [json-glib-1.0])
 
 PKG_CHECK_MODULES(APPSTREAM_GLIB, [appstream-glib >= 0.5.10])
+appstream_glib_soupapiversion=`$PKG_CONFIG --variable soupapiversion appstream-glib`
+if test "x$appstream_glib_soupapiversion" = "x"; then
+   appstream_glib_soupapiversion="2.4"
+fi
+
+if test "x$appstream_glib_soupapiversion" != "x$SOUPAPIVERSION"; then
```

Comment:
I suppose this needs to be dropped now that the development branch of Flatpak [no longer uses appstream-glib](https://github.com/flatpak/flatpak/commit/9c059def69a9f969e87e5303c2397bda18f1f883). Now we use libappstream, which recently moved (back) from libsoup to cURL (commit: https://github.com/ximion/appstream/commit/124b4c95e4f1577368077332769897cf67be7dd6).

Reply:
Sure, I can remove it. The above discussion(s) should be sorted out first, thus I can update the merge request with something final, rather than something partial.

Reply:
Removed and rebased against the current master.